package.json

```diff
@@ -16,138 +16,57 @@
   "engines": {
     "vscode": "^1.21.0"
   },
-  "categories": ["Formatters", "Programming Languages", "Linters", "Snippets"],
-  "keywords": ["ocaml", "reason", "bucklescript", "reasonml", "merlin"],
+  "categories": [
+    "Formatters",
+    "Programming Languages",
+    "Linters",
+    "Snippets"
+  ],
+  "keywords": [
+    "ocaml",
+    "reason",
+    "bucklescript",
+    "reasonml",
+    "merlin"
+  ],
   "icon": "assets/logo.png",
-  "activationEvents": ["onLanguage:ocaml", "onLanguage:reason"],
+  "activationEvents": [
+    "onLanguage:ocaml",
+    "onLanguage:reason"
+  ],
   "main": "./out/src/extension",
   "contributes": {
-    "commands": [
-      {
-        "command": "reason.caseSplit",
-        "title": "Reason: Case Split"
-      },
-      {
-        "command": "reason.showMerlinFiles",
-        "title": "Reason: Show Merlin Files"
-      },
-      {
-        "command": "reason.showAvailableLibraries",
-        "title": "Reason: Show Libraries Available via Dependencies"
-      },
-      {
-        "command": "reason.showProjectEnv",
-        "title": "Reason: Show Environment"
-      }
-    ],
     "configuration": {
       "type": "object",
       "title": "Reason configuration",
       "properties": {
-        "reason.codelens.unicode": {
-          "type": "boolean",
-          "default": true,
-          "description": "Enable the use of unicode symbols in codelens."
-        },
-        "reason.codelens.enabled": {
-          "type": "boolean",
-          "default": false,
-          "description": "Specifies whether the code lens is shown."
-        },
-        "reason.debounce.linter": {
-          "oneOf": [
-            {
-              "type": "integer"
-            },
-            {
-              "enum": ["Infinity"]
-            }
-          ],
-          "default": 500,
-          "description":
-            "How long to idle (in milliseconds) after keypresses before refreshing linter diagnostics. Smaller values refresh diagnostics more quickly."
-        },
-        "reason.diagnostics.tools": {
-          "type": "array",
-          "items": {
-            "enum": ["merlin", "bsb"]
```

Comment:
@rusty-key @andreypopp One of the main advantages of this bsb integration was that bsb has full project visibility. So if one file changes and that change breaks other files, bsb is able to show those errors in the Diagnostics panel, even if the file is not opened. One has just to click on them to open the broken file, which is useful for large refactors.

Is something like that possible with merlin? I always believed that merlin only has a "local view" of the world. 🤔